File: 프로그래머스/lv1/12950. 행렬의 덧셈/행렬의 덧셈.swift
func solution(_ arr1:[[Int]], _ arr2:[[Int]]) -> [[Int]] {
    var result : [[Int]] = []
    for i in 0..<arr1.count {
        var array : [Int] = []
        for j in 0..<arr1[0].count {
            array.append(arr1[i][j] + arr2[i][j])
        }
        result.append(array)
    }
    return result
}

File: 프로그래머스/lv1/12950. 행렬의 덧셈/행렬의 덧셈.py
def solution(arr1, arr2):
    a = []
    for i in range(len(arr1)):
        b = []
        for j in range(len(arr1[0])):
            b.append(arr1[i][j] + arr2[i][j])
        a.append(b)
    return a